src/main/java/xonin/backhand/Backhand.java

```diff
@@ -17,13 +19,14 @@
 import cpw.mods.fml.common.event.FMLServerStoppingEvent;
 import xonin.backhand.packet.BackhandPacketHandler;
 import xonin.backhand.utils.BackhandConfig;
+import xonin.backhand.utils.BackhandConfigClient;
 
 @Mod(
     modid = Backhand.MODID,
     name = "Backhand",
     version = Tags.VERSION,
     dependencies = "required-after:gtnhlib@[0.3.2,)",
-    guiFactory = "xonin.backhand.client.gui.BackhandGuiFactory")
+    guiFactory = "backhand.client.BackhandGuiFactory")
```

Comment:
If the custom factory is needed, this seems to be missing the `xonin` package at the front

Reply:
Works fine for me in game

Reply:
No longer needed